### PR TITLE
Flip serialization of two font-variation-settings tests

### PR DESCRIPTION
**Pull request by csnardi**
The expected computed valid of two font variation settings tests was opposite to the specified order (and per https://drafts.csswg.org/css-fonts-4/#propdef-font-variation-settings, the computed value is "as specified"). Reverse the two axis in order to match the specified order.

### CODE REVIEW

**Comment by csnardi** [comment]
@FremyCompany Can you review? Unless I'm missing something obvious, I can't see why the other serialization would be correct per the spec.

File: css/css-fonts/variations/font-variation-settings-parsing.html [hunk 1]
```diff
@@ -35,6 +35,7 @@
         valueParserTests.forEach(function (testCase) {
             test(() => {
                 var element = document.getElementById("value-parser-test");
+                element.style.fontVariationSettings = "";
```
Comment (drott) [inline css/css-fonts/variations/font-variation-settings-parsing.html]: Why is this needed?

**Review by drott — COMMENTED** [review]
> was opposite to the specified order.

Can you add a spec reference to the commit msg explaining the serialization order?

File: css/css-fonts/variations/font-variation-settings-parsing.html (hunk 1 above)
Comment (csnardi) [reply css/css-fonts/variations/font-variation-settings-parsing.html]: The last two failures in https://crbug.com/809950 were not actual failures -- they failed because the last test failed (leading to no resetting of font-variation-settings). Since they were invalid parsing tests, they relied on font-variation-settings not being set prior to the test. While the first commit fixes the test for Chrome, this would fix it for other browsers which might run into the same problem where one test fails immediately before an invalid parsing test.

Comment (drott) [reply css/css-fonts/variations/font-variation-settings-parsing.html]: Thanks for the clarification and finding that out. Perhaps it's useful adding a comment here, like: "Reset to empty in order to allow testing to continue in case of parse errors occuring for the next test value."

Comment (csnardi) [reply css/css-fonts/variations/font-variation-settings-parsing.html]: Good suggestion, I added that.

**Review by drott — APPROVED** [review]
Thanks for the quick changes, LGTM.

**Comment by csnardi** [comment]
@drott Could you also merge this? I unfortunately do not have write access to the repo.

**Comment by emilio** [comment]
@csnardi feel free to reach to me if you need to get something merged. I cannot grant you access, but I sure can merge your patches.

Thanks for all the work you do!

**Comment by csnardi** [comment]
@emilio Thank you, will do! :)

**Comment by FremyCompany** [comment]
I think the spec should be updated, because the serialization of the property is not "as specified". I just took a look at the exact text we introduced:

> If the same axis name appears more than once, the value associated with the last appearance supersedes any previous value for that axis. This deduplication is observable by accessing the computed value of this property.

and I agree it doesn't say you should reorder the axis names, so I'm fine with the change.

**Review by FremyCompany — COMMENTED** [review]
LGTM

**Comment by csnardi** [comment]
@FremyCompany In the property definition box, it does say "Computed value: as specified", so I believe that would apply?
